django/contrib/admin/actions.py

```diff
@@ -33,8 +33,11 @@ def delete_selected(modeladmin, request, queryset):
 
     # Populate deletable_objects, a data structure of all related objects that
     # will also be deleted.
-    deletable_objects, model_count, perms_needed, protected = get_deleted_objects(
-        queryset, opts, request.user, modeladmin.admin_site, using)
+    deletable_objects, model_count, perms_needed, protected = (
```

Comment:
I assumed this would be the expected behaviour (i.e. that deleting from the delete view or using the delete selected action on the same ModelAdmin would do the same thing in terms of the related objects they allow to be deleted) but I guess it is not immediately obvious that the two use the same method, especially since `delete_view()` and `get_deleted_objects()` are ModelAdmin methods but `delete_seleted()` is not.